root/components/Aliases/index.js

```diff
@@ -15,14 +15,30 @@ import EntityLink from '../../static/scripts/common/components/EntityLink';
 import entityHref from '../../static/scripts/common/utility/entityHref';
 import AliasTable from './AliasTable';
 
+function canEdit($c: CatalystContextT, entityType: string) {
+  if ($c.user) {
+    switch (entityType) {
+      case 'area':
+        return $c.user.is_location_editor;
+      case 'instrument':
+        return $c.user.is_relationship_editor;
+      default:
+        return !$c.user.is_editing_disabled;
```

Comment:
I think it'd be more logically correct to `return true;` here and change the `if` to `if ($c.user && !$c.user.is_editing_disabled) {`.